modules/core/src/hal_replacement.hpp

```diff
@@ -843,6 +843,30 @@ inline int hal_ni_minMaxIdx(const uchar* src_data, size_t src_step, int width, i
 #define cv_hal_minMaxIdx hal_ni_minMaxIdx
 //! @endcond
 
+/**
+   @brief calculates the mean and the standard deviation of array elements independently for each channel
+   @param src_data Source image
+   @param src_step Source image
+   @param width Source image dimensions
+   @param height Source image dimensions
+   @param src_type Type of source image
+   @param mean_needed Mean value should be computed.
+   @param mean_val Array of per-channel mean values. Up to 4 elements.
+   @param stddev_needed Stddev value should be computed.
+   @param stddev_val Array of per-channel standard deviation values. Up to 4 elements.
+   @param mask Specified array region.
+   @param mask_step Mask array step.
+   @sa meanStdDev
+*/
+inline int hal_ni_meanStdDev(const uchar* src_data, size_t src_step, int width, int height,
+                             int src_type, bool mean_needed, double* mean_val,
```

Comment:
why not just use nullptr to identify that mean_val or stddev_val are not needed?